src/samVisualize/rendering/configConstants/PhysicsConfig.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * Configuration constants regarding physical simulation
+ */
+export class PhysicsConfig {
+    /**
+     * Determines the target distance which link forces push or pull to reach between a pair of connected Nodes.
+     * The strength of the force is proportional to the difference between the linked nodes’ distance and the target distance, similar to a spring force.
+     */
+    public static readonly linkForceDistance = 200
```

Comment:
Can we put a link to the official doc here so we can quickly dive deep to each parameter?

Reply:
Good idea